cmd/telepilotd/main.go

```diff
@@ -21,14 +22,17 @@ import (
 func main() {
 	keyDir := flag.String("certs", "./certs",
 		"Certs directory. Expecting <certdir>/ca.pem, <certdir>/server.pem and <certdir>/server-key.pem.")
+	flag.Parse()
+
 	tlsConfig, err := tlsconfig.LoadTLSConfig(
 		path.Join(*keyDir, "server.pem"),
 		path.Join(*keyDir, "server-key.pem"),
 		path.Join(*keyDir, "ca.pem"),
 		false,
 	)
 	if err != nil {
-		log.Fatalf("Failed to load tls config from %q: %s.", *keyDir, err)
+		slog.Error("Failed to load tls config.", slog.String("cert_dir", *keyDir), slog.Any("error", err))
```

Comment:
FYI, we considered whether to use the slog.Attr wrappers but decided to go with the "free form" approach, except from certain key scenarios where the Attr actually saves a needless evaluation.

Reply:
Done